### PR TITLE
Add removeItemsFromArray test

### PR DESCRIPTION
**Pull request by William-Liang808**

### CODE REVIEW

File: src/test/javascript/general.controller.test.js [hunk 1]
```diff
@@ -209,12 +209,12 @@ describe("GeneralController", function () {
         });
         it("should set selectedGrouping to 'zzzz'", () => {
             scope.selectedGrouping = 0;
-           scope.displayGrouping(0, 0);
-           expect(scope.selectedGrouping).toEqual('zzzz');
+            scope.displayGrouping(0, 0);
```
Comment (Kobeyarai808) [inline src/test/javascript/general.controller.test.js:212]: Why was a space added in front of these lines?

File: src/test/javascript/general.controller.test.js [hunk 2]
```diff
@@ -381,6 +381,15 @@ describe("GeneralController", function () {
         });
     });
 
+    // describe("closeApiError", () => {
```
Comment (Kobeyarai808) [inline src/test/javascript/general.controller.test.js:384]: Is there a good reason for commenting this out? If so, please document the reason in the comment.

Comment (Kobeyarai808) [reply src/test/javascript/general.controller.test.js:384]: Same thing for the other sections which are commented out.

**Review by Kobeyarai808 — CHANGES_REQUESTED** [review]
Please take a look at the comments I made, should be very easy fixes.

File: src/test/javascript/general.controller.test.js (hunk 1 above)
Comment (William-Liang808) [reply src/test/javascript/general.controller.test.js:212]: weird, I didnt make these changes explicitly, don't know what went wrong here but I'll fix it.

File: src/test/javascript/general.controller.test.js (hunk 2 above)
Comment (William-Liang808) [reply src/test/javascript/general.controller.test.js:384]: Gotcha, idk why this is even here

**Comment by devgav** [comment]
These changes look good. I can squash and merge these commits

**Pull request merged**